plugins/inventory/proxmox.py

```diff
@@ -31,6 +31,7 @@
         description:
           - URL to Proxmox cluster.
           - If the value is not specified in the inventory configuration, the value of environment variable C(PROXMOX_URL) will be used instead.
+          - You can also use templating to specify the value of the url like "{{ lookup('ini', 'url', section='proxmox', file='file.ini') }}"
```

Comment:
```suggestion
          - Since community.general 4.7.0 you can also use templating to specify the value of the URL."
```
Better show an example in the `EXAMPLES`.

(Same for the other two options.)

Reply:
Added an example